config/environments/development.rb

```diff
@@ -87,8 +87,7 @@
   # Otherwise, use letter_opener, which launches a browser window to view sent mail.
   config.action_mailer.delivery_method = ENV['HEROKU'] || ENV['VAGRANT'] || ENV['REMOTE_DEV'] ? :letter_opener_web : :letter_opener
 
-  # We provide a default secret for the development environment here.
-  # This value should not be used in production environments!
+  # TODO: Remove once devise-two-factor data migration complete
```

Comment:
We need to define when we remove this. As we support multiple version upgrades (4.1 => 4.4), should this be for Mastodon 5.0, and we ask users to first update to the latest 4 version before updating to 5.0?

(same for the other places where we have this TODO)

Reply:
It is not, but it would probably be a good thing to do it.